tests/integration_python/test_edge_cases.py

```diff
@@ -4,7 +4,7 @@
 from .common import verify_cli_command, ExitCode
 
 
-@pytest.mark.slow
+@pytest.mark.extra_slow
```

Comment:
should we record this new marker?
https://docs.pytest.org/en/stable/example/markers.html#registering-markers